installation/README.md

```diff
@@ -0,0 +1,87 @@
+# Installing Config Sync
+
+This document is a guide on how to install Config Sync on a kubernetes cluster.
+
+This document includes configurations for some common user journeys, but is not
+an exhaustive list of how a Config Sync installation can be customized.
+
+## Pre-requisites
+
+This guide requires the following command line tools:
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+- [kustomize](https://github.com/kubernetes-sigs/kustomize)
+
+## Uninstall Config Management
```

Comment:
Uninstall Config Management with other components

Reply:
Why "with other components"?

Reply:
`other components` refers to HC. Now we added it as a prerequisite, we can leave it as-is.